Project not compiling due to unused code

The project cannot be built due to the following errors:

```rust
error: multiple fields are never read
  --> tests/common/contexts/torrent/file.rs:13:9
   |
12 | pub struct TorrentFileInfo {
   |            --------------- fields in this struct
13 |     pub name: String,
   |         ^^^^
...
18 |     pub source: Option<String>,
   |         ^^^^^^
19 |     pub info_hash: String,
20 |     pub torrent_size: u64,
   |         ^^^^^^^^^^^^
21 |     pub content_size: u64,
22 |     pub private: bool,
   |         ^^^^^^^
23 |     pub tracker: Option<String>,
   |         ^^^^^^^
24 |     pub announce_list: Option<Vec<Vec<String>>>,
   |         ^^^^^^^^^^^^^
25 |     pub update_url: Option<String>,
   |         ^^^^^^^^^^
26 |     pub dht_nodes: Option<Vec<String>>,
   |         ^^^^^^^^^
27 |     pub piece_size: u64,
   |         ^^^^^^^^^^
28 |     pub piece_count: u64,
   |         ^^^^^^^^^^^
29 |     pub file_count: u64,
   |         ^^^^^^^^^^
   |
   = note: `TorrentFileInfo` has derived impls for the traits `Debug` and `Clone`, but these are intentionally ignored during dead code analysis
   = note: `-D dead-code` implied by `-D unused`
   = help: to override `-D unused` add `#[allow(dead_code)]`

error: fields `title`, `description`, `category`, and `torrent_file` are never read
  --> tests/common/contexts/torrent/fixtures.rs:45:9
   |
43 | pub struct TorrentListedInIndex {
   |            -------------------- fields in this struct
44 |     pub torrent_id: Id,
45 |     pub title: String,
   |         ^^^^^
46 |     pub description: String,
   |         ^^^^^^^^^^^
47 |     pub category: String,
   |         ^^^^^^^^
48 |     pub torrent_file: BinaryFile,
   |         ^^^^^^^^^^^^

error: field `data` is never read
 --> tests/common/contexts/user/responses.rs:5:9
  |
4 | pub struct AddedUserResponse {
  |            ----------------- field in this struct
5 |     pub data: NewUserData,
  |         ^^^^
  |
  = note: `AddedUserResponse` has a derived impl for the trait `Debug`, but this is intentionally ignored during dead code analysis

error: field `user_id` is never read
  --> tests/common/contexts/user/responses.rs:10:9
   |
9  | pub struct NewUserData {
   |            ----------- field in this struct
10 |     pub user_id: i64,
   |         ^^^^^^^
   |
   = note: `NewUserData` has a derived impl for the trait `Debug`, but this is intentionally ignored during dead code analysis

error: could not compile `torrust-index` (test "mod") due to 4 previous errors
Error: Process completed with exit code 101.
```

--- a/tests/common/contexts/torrent/file.rs
+++ b/tests/common/contexts/torrent/file.rs
@@ -8,6 +8,7 @@ use serde::Deserialize;
 use which::which;
 
 /// Attributes parsed from a torrent file.
+#[allow(dead_code)]
 #[derive(Deserialize, Clone, Debug)]
 pub struct TorrentFileInfo {
     pub name: String,

--- a/tests/common/contexts/torrent/fixtures.rs
+++ b/tests/common/contexts/torrent/fixtures.rs
@@ -40,6 +40,7 @@ impl From<TorrentIndexInfo> for UploadTorrentMultipartForm {
 }
 
 /// Torrent that has been added to the index.
+#[allow(dead_code)]
 pub struct TorrentListedInIndex {
     pub torrent_id: Id,
     pub title: String,

--- a/tests/common/contexts/user/responses.rs
+++ b/tests/common/contexts/user/responses.rs
@@ -1,10 +1,12 @@
 use serde::Deserialize;
 
+#[allow(dead_code)]
 #[derive(Deserialize, Debug)]
 pub struct AddedUserResponse {
     pub data: NewUserData,
 }
 
+#[allow(dead_code)]
 #[derive(Deserialize, Debug)]
 pub struct NewUserData {
     pub user_id: i64,

